src/AFQMC/Estimators/FullObsHandler.hpp

```diff
@@ -111,20 +111,20 @@ class FullObsHandler : public AFQMCInfo
       std::transform(cname.begin(), cname.end(), cname.begin(), (int (*)(int))tolower);
       if (cname == "onerdm")
       {
-        properties.emplace_back(Observable(std::move(full1rdm(TG, info, cur, walker_type, nave, block_size))));
+        properties.emplace_back(Observable(full1rdm(TG, info, cur, walker_type, nave, block_size)));
```

Comment:
compilers support copy elision(optional as an optimization prior C++17 and mandatory since C++17) will construct full1rd inplace into Observable.
If the compiler doesn't support copy elision, it first construct the temporal object (prvalue) and then invoke the move constructor which is the old code does.
I don't think it hurts us in C++14.